docs/api/session.md

```diff
@@ -447,7 +447,8 @@ the original network configuration.
     * `hostname` String
     * `certificate` [Certificate](structures/certificate.md)
     * `validatedCertificate` [Certificate](structures/certificate.md)
-    * `verificationResult` String - Verification result from chromium.
+    * `isIssuedByKnownRoot` Boolean - `true` if Chromium recognises the root CA as a standard root. If it isn't then it's probably the case that this certificate was generated by a MITM proxy whose root has been installed locally. This is meaningless if the certificate was not trusted.
```

Comment:
```suggestion
    * `isIssuedByKnownRoot` Boolean - `true` if Chromium recognises the root CA as a standard root. If it isn't then this certificate's root has been installed locally, for example by a corporate proxy. This is meaningless if the certificate was not trusted.
```

Also, I'm not sure what "if the certificate was not trusted" here means. Something about `verificationResult`?

Reply:
@nornagon I just copied it from Chromium, but yes. How about this:

```suggestion
    * `isIssuedByKnownRoot` Boolean - `true` if Chromium recognises the root CA as a standard root. If it isn't then it's probably the case that this certificate was generated by a MITM proxy whose root has been installed locally (for example, by a corporate proxy). This should not be trusted if the `verificationResult` is not `'OK'`.
```